doc/support.rst

```diff
@@ -11,6 +11,11 @@ Time Integration Loops
 
 .. automodule:: mirgecom.steppers
 
+MPI
```

Comment:
In `running.rst`, document that `-m mpi4py` is the preferred way to run.

Reply:
Is this already done?

Reply:
I don't think it's explicitly stated anywhere, but it's shown that way in the run script examples, so I figured that was good enough. Also, with this change`-m mpi4py` will become mandatory for running MPI examples (see other comment).

Reply:
Ok, in any case we can add this in another PR.